playground/mocks/data/idp/idx/safe-mode-credential-enrollment-intent.json

```diff
@@ -11,9 +11,9 @@
     "type": "array",
     "value": [
       {
-        "message": "Set up is temporarily unavailable due to server maintenance. Click 'Back to Settings' to continue.",
+        "message": "Set up is temporarily unavailable due to server maintenance. Try again later.",
         "i18n": {
-          "key": "idx.error.server.safe.mode.cred.enroll"
+          "key": "idx.error.server.safe.mode.enrollment.unavailable"
```

Comment:
Should have mentioned this in the PR description, but all safe-mode errors include a single i18n key. You can see the server-side implementation [here](https://github.com/okta/okta-core/pull/47490/files).